README.md

```diff
@@ -32,21 +32,21 @@ the user would have with other frameworks:
 * [x] Supports parameterized tests and theories.
 * [x] Progress and statistics can be followed in real time with report hooks.
 * [x] TAP output format can be enabled with an option.
-* [x] Runs on Linux, FreeBSD, Mac OS X, and Windows (Compiling with MinGW GCC and Visual Studio 2015+).
+* [x] Runs on Linux, FreeBSD, macOS, and Windows (Compiling with MinGW GCC and Visual Studio 2015+).
 
 ## Downloads
 
 ### Packages
 
-* Mac OS X: `brew install snaipe/soft/criterion`
-* [AUR](https://aur.archlinux.org/packages/criterion/): `pacaur -S criterion`
-* Ubuntu/Debian:
+* Ubuntu/Debian: `apt-get install libcriterion-dev` or PPA:
```

Comment:
Could you add the minimum version of Ubuntu and Debian at which Criterion is installable?

Reply:
Minimum Ubuntu/Debian versions have been added.